modules/auxiliary/scanner/ssh/ssh_identify_pubkeys.rb

```diff
@@ -84,19 +90,19 @@ def read_keyfile(file)
       keyfile = datastore['SSH_KEYFILE_B64'].unpack("m*").first
     elsif file.kind_of? Array
       keyfile = ''
-      file.each do |dir_entry|
+      file.map { |f| ::File.expand_path(f) }.each do |dir_entry|
```

Comment:
I believe this is now duplicated and unnecessary given my changes in lib/msf/core/opt_path.rb